components/ArtPieces/index.js

```diff
@@ -1,3 +1,22 @@
+import { StyledContainer } from "./ArtPieces.styled"
+import ArtPiecePreview from "../ArtPiecePreview";
+
 export default function ArtPieces({ pieces }) {
-  return <div>Art Pieces</div>;
+  return( 
+  <StyledContainer>
+    {pieces && pieces.length > 0 ? (pieces.map((piece) => (
```

Comment:
No need for checking `pieces.length > 0`. Empty array => [] will not cause a problem for map function. It will just not change anything.